### DISCUSSION TITLE
Db connection

### DISCUSSION THREAD

**Discussion by nezzard** [vercel/next.js#32185] [Help]
Hi, next js 12 is awesome. 
I have one question, am I correct that theoretically I can remove backend from project and put db connection (for example mongo cloud) to api files, and it’ll be secure? 
sorry for my bad English

**Comment by 1finedev** [comment]
That might be bad for you if you intend to reuse the DB connection
if i am correct an api route must return to terminate correctly also the api routes must end whatever its doing after some seconds
